packages/cascara/src/ui/Form/context/FormProvider.js

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable-next-line eslint-comments/disable-enable-pair -- we need this for the one below */
+/* eslint-disable jsx-a11y/no-redundant-roles -- we need this for form to be accessible */
+
```

Comment:
Why are we disabling this lint rule? Some tags have native semantics which do not require redundantly refining the role.

Reply:
You are correct on the fact that From has an instrinsic role of `form`, But there is a bug in RTL in which Forms do not pop-out when using `ByRole` queries - This was just an attempt to solve the issue by manually adding the role...

Reply:
It seems like [the issue is coming from us not defining a `name` or `aria-labeledby`](https://github.com/testing-library/dom-testing-library/issues/474) on the Form. We should investigate, but this is probably the route we should take instead of disabling the eslint rule. Could you check if adding either of those parameters make the Form show up using that RTL selector?

Worst case scenario though... we almost never want to be disabling an eslint rule for the whole file. We usually only want to be disabling it for next line instead.

Reply:
This has been removed, we found another way for querying the `form` and focus the test to it.